### PR TITLE
[OPA-1278] GitEx for OPA

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by dwarren-dev — APPROVED** [review]

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+You can store your policies in one of the following Git providers: 
+- GitHub
+- Bitbucket Cloud
+- Bitbucket Server
+- Azure Repos
+- GitLab
+
+## Important
+
```
Comment (StrawberryPad) [inline docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:25]:
```suggestion
To create a remote policy, you must have:
```

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- To create remote policies, you must have **Create/Edit** permissions for policies.
```
Comment (StrawberryPad) [inline docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:26]:
```suggestion
- **Create/Edit** permissions for policies.
```

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Make sure you have a Git repo with at least one branch.
```
Comment (StrawberryPad) [inline docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:27]:
```suggestion
- A Git repo with at least one branch.
```

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- Make sure you have a Git connector with a Personal Access Token (PAT) for your Git account.​​
```
Comment (StrawberryPad) [inline docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:28]:
```suggestion
- A Git connector with a Personal Access Token (PAT) for your Git account.​​
```

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- A Personal Access Token (PAT) for your Git account.
```
Comment (StrawberryPad) [inline docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:29]:
```suggestion
- A PAT for your Git account.
```

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- Harness needs the PAT to use the Git platform APIs.
+- You add the PAT to Harness as an [encrypted text](../../Secrets/2-add-use-text-secrets.md), and it is encrypted using a Harness Secret Manager.
+- Your Git Personal Access Token is stored in your Harness secret and is a private key to which only you have access. This secret cannot be accessed or referenced by any other user.
+
```
Comment (StrawberryPad) [inline docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:33]: The above 3 bullets would be sub-bullets (indented) under "A PAT for your Git account". My suggestions above make this a list of 4 prereq's, with the 4th one having 3 sub-bullets.

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+## Create a remote policy
+
+To create a remote policy in Harness: 
+
```
Comment (StrawberryPad) [inline docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:37]: Suggest combining steps 1 + 2, and 3 + 4 below just to make the steps a bit more compact and easier to scroll through (and appear simpler to the user).

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+1. Select **Policies**.
+2. Select **New Policy**.
+3. Enter a name for your policy.
+4. Select **Remote**. The additional settings appear to configure Git Experience.
+5. In **Git Connector**, select or create a Git Connector to the repo for your Project. For steps, see [Code Repo Connectors](../../7_Connectors/Code-Repositories/connect-to-code-repo.md).
+6. In **Repository**, select your repository. If your repository isn't listed, enter its name since only a select few repositories are filled here.
+7. In **Git Branch**, select your branch. If your branch isn't listed, enter its name since only a select few branches are filled here.
+8. Harness pre-populates the **Rego Path**. You can change this path and the file name.
```
Comment (StrawberryPad) [inline docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:45]: This isn't a step instruction, but seems like a sub-paragraph (indented) of Step 7.

**Review by StrawberryPad — CHANGES_REQUESTED** [review]
Nice! At a minimum: there's a typo (period at end of sentence), and suggested changes to the "Important" heading. That heading title should be avoided since it's non-specific and doesn't really describe what's under it. "Prerequisites" or "Requirements" would be much better.

File: docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md (hunk 8 above)
Comment (rashminandasahoo) [reply docs/platform/Governance/14_Policy-as-code/configure-gitexperience-for-opa.md:45]: Rego Path is a separate field but is prepopulated. Customers can change it. So I will retain it as a separate step.

**Review by StrawberryPad — APPROVED** [review]

**Pull request merged**